mobile-app/src/atoms/atoms.ts

```diff
@@ -582,3 +583,16 @@ export const settingAtom = atomFamily((settingKey: string) =>
     { getOnInit: true },
   ),
 );
+
+export const profileFormDataAtom = atomWithDefault((get) => {
+  return { username: get(currentUserAtom).username, displayName: get(currentUserAtom).displayName }
+}
+)
+
+profileFormDataAtom.onMount = (setAtom) => {
```

Comment:
what u trying to do here?

Reply:
This is more of a debug thing, its kinda useful tho so im not 100% sure i wanna delete it